src/here-xyz.ts

```diff
@@ -362,6 +346,52 @@ async function listSpaces(options: any) {
     }
 }
 
+
+/**
+ * Will list all the projects for the given user in below format
+ *
+ * @param options
+ */
+async function listProjects (options: any) {
```

Comment:
if this function is not used, please remove the unused code

Reply:
This is now removed, was left as previously was used in ```here xyz list``` command. Forgotten to update.